package.json

```diff
@@ -5,12 +5,12 @@
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "config": {
-    "lnd_release_tag": "v0.18.2-beta",
+    "lnd_release_tag": "v0.18.4-beta",
     "loop_release_tag": "v0.28.5-beta",
```

Comment:
Should we also bump this to `v0.29.0-beta` to match LNC?

Reply:
I updated the Loop version to `v0.29.0-beta`.